public_html/autoload.php

```diff
@@ -11,4 +11,7 @@
  * @see core/modules/statistics/statistics.php
  */
 
+// commented out code for sonarqube to complain
+// $this = 'that';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.jb.dkel7.jellyfish.local/coding_rules#rule_key=php%3AS125)